build.yaml

```diff
@@ -1002,13 +1002,16 @@ run_trial_modules: &run_trial_modules
   - flocker.test
   - flocker.testtools
 run_trial_as_root_modules: &run_trial_as_root_modules
+  # iptables requires root
+  - flocker.route.functional
+  # journald access requires root
+  - flocker.common.functional
   # flocker.volume needs to run as root due to ZFS calls
   # so we run it as part of the run_trial_on_<Cloud>_<OS>_as_root jobs
   - flocker.volume
   # flocker.node.agent needs to run as root due to mounting filesystems
   - flocker.node.agents
 
```

Comment:
There's e.g. `flocker.common.test` too, which doesn't need root. So some minor duplication but skips are fast, typically, so not a big deal.